charts/openmetadata/templates/hpa.yml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.hpa.enabled -}}
+apiVersion: autoscaling/v2
```

Comment:
Hello @rishi-opensource . Let's get this from `.Values.hpa.apiVersion` instead! A lot of times, the Kubernetes API Versions get matured and we want users to be flexible of choosing their own API Version for HPA based on the existing kubernetes Version.

Reply:
Yeah, that makes sense. I've made the changes accordingly.